src/shared/components/CSVExportButton.tsx

```diff
@@ -59,7 +64,27 @@ class CSVExportButton extends PureComponent<Props, State> {
       ).then(
         registrationResult => {
           if (registrationResult.active.state == 'activated') {
-            this.setState({browserSupportsDownloadInflxQL: true})
+            this.setState({browserSupportsDownloadInfluxQL: true})
+          } else {
+            console.error(
+              'Feature not available, because ServiceWorker is registered but inactive.'
+            )
+          }
+        },
+        function (err) {
+          console.error(
+            'Feature not available, because ServiceWorker registration failed: ',
+            err
+          )
+        }
+      )
+    }
+
+    if (isFlagEnabled('v2privateQueryUI') && props.workerRegistrationSQL) {
+      ;(props.workerRegistrationSQL as Promise<ServiceWorkerRegistration>).then(
```

Comment:
I'm not very familiar with the ;() syntax here. Would you be able to explain what it's doing?

Reply:
I also had this question before! It's automatically added by prettier when I save the file. It's a wired case and here is an explanation: https://github.com/influxdata/ui/pull/5270#discussion_r936974286